test/k8sT/Updates.go

```diff
@@ -207,8 +210,38 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 			helpers.KubeSystemNamespace, l7Policy, helpers.KubectlApply, timeout)
 		Expect(err).Should(BeNil(), "cannot import l7 policy: %v", l7Policy)
 
+		_, err = kubectl.CiliumPolicyAction(
+			helpers.KubeSystemNamespace, l4Policy, helpers.KubectlApply, timeout)
+		Expect(err).Should(BeNil(), "cannot import l7 policy: %v", l7Policy)
+
 		validateEndpointsConnection()
 
+		By("Starting background process that checks L4 connectivity each 5 seconds")
```

Comment:
1. Define a const the interval
2. Change the interval to 0.5 seconds
3. Use the const here and in the ticker below

Reply:
@tgraf Shouldn't the test to be a TCP connection and always send traffic from client to the server and check if all information sent was also received??

Reply:
Both are valuable. A persistent connection can also hide some defects. I'm fine if what Eloy proposes is what we start out with.